docs/docs/api/LayoutAnimations/EntryAnimations.md

```diff
@@ -0,0 +1,83 @@
+---
+id: entryAnimations
+title: Entering Animations
+sidebar_label: Entering Animations
+---
+
+## How to use predefined entering animation?
+
+### 1. Import chosen animation
+```js
+    import { Animation } from 'react-native-reanimated';
+```
+### 2. Choose Animated Component which entering you want to animate
+```js
+    <AnimatedComponent entering={animation} >
```

Comment:
I think it should be `Animation` instead of `animation` because confuses a bit :)

Reply:
Nice catch! Thanks